openstacksdk_plugin/tests/test_network.py

```diff
@@ -24,9 +24,12 @@
     security_group,
     security_group_rule,
     floating_ip)
+from openstacksdk_plugin.constants import RESOURCE_ID
 
 from cloudify.state import current_ctx
 
+MODULE_PATH = 'openstack.cloud.openstackcloud._OpenStackCloudMixin'
+
 
 @mock.patch('openstack.connect')
 class TestNetwork(OpenStackTestBase):
```

Comment:
@EarthmanT what do you think about changing this name to `NetworkTestCase` in order to match with other and to avoid using verbs at the beginning of the classes

Reply:
OK sure.